data_steward/curation_logging/test_main.py

```diff
@@ -0,0 +1,22 @@
+"""
+Notes: -- if dev is using macOS and gets error:
```

Comment:
`test_main.py` was something I created for easy testing, however, this script may not belong to the repository and this is like a precursor of the unit test.  I do love your notes here. I am wondering if you could move the notes to `slack_logging_handler.py` or `slack_logging_handler_test.py`

Reply:
I'm not sure it would make sense to put the notes in those places as the logging messages aren't being sent to the slack channel. Would it be beneficial to create another test function to send them to the test slack channel? We would need to give everyone access to that test channel, will thee same slack token work for everyone?